packages/react-component-library/src/components/Timeline/TimelineRows.tsx

```diff
@@ -12,7 +12,7 @@ export interface TimelineRowsProps extends ComponentWithClass {
   children:
     | React.ReactElement<TimelineRowProps>
     | React.ReactElement<TimelineRowProps>[]
-  renderColumns?: (
+  render?: (
```

Comment:
👮 

This is a breaking change..?

Reply:
Oops yes i'll revert that

Reply:
I've created a ticket for this breaking change and labelled it as V3: https://github.com/Royal-Navy/design-system/issues/1199